pkgs/os-specific/linux/udisks/2-default.nix

```
@@ -35,6 +35,8 @@
   nilfs-utils,
   ntfs3g,
   nixosTests,
+  libiscsi,
+  libconfig
```

```suggestion
  libconfig,
```
To fix the formatter check.